renderer.js

```diff
@@ -174,3 +229,88 @@ function getSongs(e) {
             }
         });
 }
+
+
+// Suggestions
+function findMatches(wordToMatch, suggestions) {
+    return suggestions.filter(suggestion => {
+        // here we need to figure out if the city or state matches what was searched
+        const regex = new RegExp(wordToMatch, 'gi');
+        return suggestion.match(regex)
+    });
+}
+
+function displayMatches() {
+    if (this.value.length < 3) {
+        return;
+    }
+    fetch(`https://suggestqueries.google.com/complete/search?client=firefox&q=${this.value}`)
+        .then(blob => blob.json())
+        .then(response => {
+            const matchArray = findMatches(this.value, response[1]);
+            const typeheadList = matchArray.map(suggestion => {
+                const regex = new RegExp(this.value, 'gi');
+                const sugesstionText = suggestion.replace(regex, `<span class="hl">${this.value}</span>`);
+                return `<li data-suggestion="${suggestion}" class="suggestionElement"><span class="name">${sugesstionText}</span></li>`;
```

Comment:
We replace the `this.value` with a `span.hl` to highlight the typed part in the suggestion.